lib/phoenix_swagger/json_api.ex

```diff
@@ -249,7 +250,7 @@ defmodule PhoenixSwagger.JsonApi do
             }
           },
           data: %Schema{
-            type: :object,
+            type: opts[:type] || :object,
```

Comment:
It didn't; I had assumed it was correct when the validations against our responses passed. It's been corrected and checked on the [Swagger Editor](http://editor.swagger.io).